site/en/docs/crux/methodology/index.md

```diff
@@ -178,17 +186,18 @@ The CrUX dataset is made available through a variety of tools maintained by Goog
 <td>Core web vitals</td>
 <td>No dimensions</td>
 <td>Three months</td>
-<td>Page Group <a href="#footnote-5"><sup>5</sup></a></td>
+<td>Page Group <a href="#footnote-6"><sup>6</sup></a></td>
 </tr>
 </tbody>
 </table></div>
 
 <p>
 <a id="footnote-1"><sup>1</sup></a> Monthly data is released on the second Tuesday after each monthly collection period. The last 28 days of each month period are included.<br>
 <a id="footnote-2"><sup>2</sup></a> 28-day rolling average data is updated daily, based on the aggregated data from the previous 28 days.<br>
-<a id="footnote-3"><sup>3</sup></a> Not all metrics are available in all monthly tables, see the <a href="/docs/crux/release-notes">release notes</a> for details.<br>
+<a id="footnote-3"><sup>3</sup></a> Weekly historical data is released every Monday, containing the previous 25-week collection periods. Each period covers a 28-day period from Sunday to Saturday.<br>
```

Comment:
... containing the previous 25-week collection periods. Each period covers a 28-day period from Sunday to Saturday.

maybe instead:

... containing the 25 most recent 28 day collection periods that end in Saturdays.